CHANGELOG.md

```diff
@@ -20,6 +20,8 @@
 
 ### Backward-incompatible changes [experimental]
 * `TabularMSA.append` and `TabularMSA.extend` now require one of `minter`, `index`, or `reset_index` to be provided when incorporating new sequences into an MSA. Previous behavior was to auto-increment the index labels if `minter` and `index` weren't provided and the MSA had a default integer index, otherwise error. Use `reset_index=True` to obtain the previous behavior in a more explicit way.
+* `skbio.stats.composition.ancom` now returns two `pd.DataFrame` objects, where it previously returned one. The first contains the ANCOM test results, as before, and the second contains percentile abundances of each feature in each group. The specific percentiles that are computed and returned is controlled by the new `percentiles` parameter to `skbio.stats.composition.ancom`. In the future, this second `pd.DataFrame` will not be returned by this function, but will be available through the [contingency table API](https://github.com/biocore/scikit-bio/issues/848). ([#1293](https://github.com/biocore/scikit-bio/issues/1293))
```

Comment:
Hold that thought.
On Jun 8, 2016 2:36 PM, "Jai Ram Rideout" <notifications@github.com> wrote:

> In CHANGELOG.md
> <https://github.com/biocore/scikit-bio/pull/1374#discussion_r66344809>:
>
> > @@ -20,6 +20,8 @@
> >
> >  ### Backward-incompatible changes [experimental]
> >  * `TabularMSA.append` and `TabularMSA.extend` now require one of `minter`, `index`, or `reset_index` to be provided when incorporating new sequences into an MSA. Previous behavior was to auto-increment the index labels if `minter` and `index` weren't provided and the MSA had a default integer index, otherwise error. Use `reset_index=True` to obtain the previous behavior in a more explicit way.
> > +* `skbio.stats.composition.ancom` now returns two `pd.DataFrame` objects, where it previously returned one. The first contains the ANCOM test results, as before, and the second contains percentile abundances of each feature in each group. The specific percentiles that are computed and returned is controlled by the new `percentiles` parameter to `skbio.stats.composition.ancom`. In the future, this second `pd.DataFrame` will not be returned by this function, but will be available through the [contingency table API](https://github.com/biocore/scikit-bio/issues/848). ([#1293](https://github.com/biocore/scikit-bio/issues/1293))
>
> Can you also note the "reject" -> "Reject null hypothesis" column name
> change in the first data frame that's returned?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/biocore/scikit-bio/pull/1374/files/e4f8c0e52a375d8db049910af2f40680e74cea36#r66344809>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AD_a3fTZS5FW0076RKC4UeNEE-lWaXeRks5qJzXagaJpZM4IwbTG>
> .
>